internal/status/checksum.go

```diff
@@ -32,6 +33,16 @@ func (c *Checksum) IsUpToDate() (bool, error) {
 		return false, err
 	}
 
+	generates, err := glob(c.Dir, c.Generates)
+	if err != nil || len(generates) == 0 {
+		return false, err
+	}
+	for _, generate := range generates {
+		if _, err := os.Stat(generate); err != nil {
```

Comment:
I think this `for` loop + check is not necessary at all. Let me know if I'm missing something.

Reply:
Yes this is unnecessary indeed. Thank you for the catch!